src/openrct2/drawing/ImageImporter.h

```diff
@@ -53,14 +54,16 @@ namespace OpenRCT2::Drawing
     private:
         static const PaletteBGRA StandardPalette[256];
 
-        static std::vector<int32_t> GetPixels(const uint8_t * pixels, uint32_t width, uint32_t height, IMPORT_FLAGS flags, IMPORT_MODE mode);
-        static std::tuple<void *, size_t> EncodeRaw(const int32_t * pixels, uint32_t width, uint32_t height);
-        static std::tuple<void *, size_t> EncodeRLE(const int32_t * pixels, uint32_t width, uint32_t height);
+        static std::vector<int32_t>
+            GetPixels(const uint8_t* pixels, uint32_t width, uint32_t height, IMPORT_FLAGS flags, IMPORT_MODE mode);
+        static std::tuple<void*, size_t> EncodeRaw(const int32_t* pixels, uint32_t width, uint32_t height);
+        static std::tuple<void*, size_t> EncodeRLE(const int32_t* pixels, uint32_t width, uint32_t height);
 
-        static int32_t CalculatePaletteIndex(IMPORT_MODE mode, int16_t * rgbaSrc, int32_t x, int32_t y, int32_t width, int32_t height);
-        static int32_t GetPaletteIndex(const PaletteBGRA * palette, int16_t * colour);
-        static bool IsTransparentPixel(const int16_t * colour);
+        static int32_t
+            CalculatePaletteIndex(IMPORT_MODE mode, int16_t* rgbaSrc, int32_t x, int32_t y, int32_t width, int32_t height);
```

Comment:
Increasing the value of `PenaltyReturnTypeOnItsOwnLine` should prevent the function line from being put on another line than the return type.

(mainly a note to self, I intend to do this after I went over the current commits)